Use new JS Solr Utils library

rderik opened the pull request:
The code has been updated to use the new JS Solr Utils library that
reduces the code and makes the partial be cleaner.

This commit contains the following changes:

Refactors code for fancyTree and collapsible plugin
The name of the plugins was changed on kmaps_engine and now the change
to those names has been reflected in this commit.
fancyTree was renamed to kmapsRelationsTree
collapsibleList was renamed to kmapsCollapsibleList

Update code to on related for summary
Cleaned code for new update on solr-utils library

rderik committed:
Use new JS Solr Utils library

The code has been updated to use the new JS Solr Utils library that
reduces the code and makes the partial be cleaner.

This commit contains the following changes:

Refactors code for fancyTree and collapsible plugin
The name of the plugins was changed on kmaps_engine and now the change
to those names has been reflected in this commit.
fancyTree was renamed to kmapsRelationsTree
collapsibleList was renamed to kmapsCollapsibleList

Update code to on related for summary
Cleaned code for new update on solr-utils library

amontano commented on app/views/features/_related.html.erb:
```diff
@@ -70,127 +38,51 @@
       // Functionality for columnizer
       // dontsplit = don't break these headers
-      $('.subjects-in-subjects').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
+      $('.places-in-places').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
```
I think the previous line "subjects-in-subjects" was correct and not places-in-places given that we are in subjects_engine!

amontano replied:
Others line below also referencing places-in-places. Check whole file.

amontano commented on app/views/features/_related.html.erb:
```diff
@@ -248,40 +139,5 @@
-}
-</style>
-<%= javascript_include_tag('collapsible_list/jquery.collapsibleList') %>
+<%= stylesheet_link_tag('subjects_engine/related') %>
```
better to have stylesheet as above as possible in this file.

rderik replied:
Move CSS tag to the top of the partial

rderik commented on app/views/features/_related.html.erb:
```diff
@@ -70,127 +38,51 @@
       // Functionality for columnizer
       // dontsplit = don't break these headers
-      $('.subjects-in-subjects').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
+      $('.places-in-places').find('.column > h6, .column > ul > li, .column ul').addClass("dontsplit");
```
Change this selector to be subjects-in-subjects,

rderik commented on app/views/features/_related.html.erb:
```diff
@@ -70,127 +38,51 @@
-        width: 330,
-        lastNeverTallest : true,
-        buildOnce: true
+      $('.places-in-places').find('.column > h6, .column > ul > li').addClass("dontend");
```
All this sleecteros form line 76-82 should use subjects not places

rderik reviewed (commented):
Make the small changes detailed on the comments before the merge

rderik committed:
Fix bugs on related partial for subjects

There were a few errors on JavaScript selectors pointing to
places instead of subjects.

Moved CSS stylesheet tag to the top of partial.
